tools/bisector/bisector/bisector.py

```diff
@@ -5138,10 +5251,21 @@ def _main(argv):
 
     SHOW_TRACEBACK = args.debug
     service_hub = ServiceHub()
+
+    artifacts_service = None
     try:
-        service_hub.register_service('upload', ArtifactorialService())
-    except Exception as e:
-        info('Artifactorial upload will not be available: ' + str(e))
+        artifacts_service = ArtifactoryService()
+        info('Artifactory Service was initialized.')
```

Comment:
Should be moved out in an `else` clause of the try/except/else statement as we only want to catch exceptions raised by `ArtifactoryService()`